galley/pkg/config/analysis/analyzers/auth/mtls/policy_checker.go

```diff
@@ -0,0 +1,224 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mtls
+
+import (
+	"fmt"
+
+	"istio.io/istio/galley/pkg/config/analysis/analyzers/util"
+	"istio.io/istio/security/proto/authentication/v1alpha1"
+)
+
+// Workload is a simple struct type for representing a service workload
+// targeted by an Authentication policy.
+type Workload struct {
```

Comment:
Policy as of release 1.3 target service by short name. Why are we calling this Workloads?

Reply:
Workloads in Istio mean pods which have the sidecar proxy and in Kubernetes map to deployments.

Reply:
I stole the 'workload' terminology from the doc on Policy https://istio.io/docs/reference/config/istio.authentication.v1alpha1/#Policy

> targets: List rules to select workloads that the policy should be applied on. If empty, policy will be used on all workloads in the same namespace.

I can see why it's called workloads as that's where the sidecar proxies are actually located. However the port you provide is the port used by the service, not the workload. Maybe it should be called ServiceWorkload?

Honestly I'm fine with Service/Workload/whatever. I'd like to know the ideal term here though so I can propose a change to the docs if it's wrong.

Reply:
That documentation is very confusing and tbh incorrect. IIRC I had opened an issue to reconcile the usage of workloads consistently, don't know if it got fixed.

I would suggest using `TargetService` as that's more inline with the current implementation.

Reply:
Renamed to TargetService. Workload is no longer found in the code anywhere.